### ISSUE TITLE
In case of toolbar, do I have to implement it myself?

### ISSUE THREAD

**Issue by brgndyy** [uiwjs/react-codemirror#564]
I know that a toolbar is provided when using other markdown-editor libraries, but do I have to implement the toolbar myself within the codemirror library?

**Comment by jaywcjlove** [comment]
@brgndyy  I implemented a https://github.com/uiwjs/react-markdown-editor

**Comment by brgndyy** [comment]
Yes I know this exists. But what I'm curious about is whether there is a separate toolbar within the codemirror library.

**Comment by jaywcjlove** [comment]
@brgndyy It is possible to customize a search-like tool, but it is very troublesome.
